apps/dashboard/package.json

```diff
@@ -4,9 +4,13 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
-    "start": "vite",
+    "start": "pnpm legacy:clean && pnpm legacy:copy && vite",
     "dev": "pnpm start",
-    "build": "tsc -b && vite build",
+    "build": "pnpm legacy:prepare && tsc -b && vite build",
+    "legacy:prepare": "pnpm legacy:clean && pnpm legacy:build && pnpm legacy:copy",
+    "legacy:clean": "rimraf legacy/*",
+    "legacy:build": "cd ../.. && pnpm build:web-for-dashboard",
```

Comment:
Since these tasks affect the web app, shall we add them to the package.json of `web`?